src/Magnum/Platform/GlfwApplication.h

```diff
@@ -0,0 +1,753 @@
+            I = 'i',                    /**< Letter I */
+            J = 'j',                    /**< Letter J */
+            K = 'k',                    /**< Letter K */
+            L = 'l',                    /**< Letter L */
+            M = 'm',                    /**< Letter M */
+            N = 'n',                    /**< Letter N */
+            O = 'o',                    /**< Letter O */
+            P = 'p',                    /**< Letter P */
+            Q = 'q',                    /**< Letter Q */
+            R = 'r',                    /**< Letter R */
+            S = 's',                    /**< Letter S */
+            T = 't',                    /**< Letter T */
+            U = 'u',                    /**< Letter U */
+            V = 'v',                    /**< Letter V */
+            W = 'w',                    /**< Letter W */
+            X = 'x',                    /**< Letter X */
+            Y = 'y',                    /**< Letter Y */
+            Z = 'z',                    /**< Letter Z */
+
+            /* Function keys */
+            Esc = GLFW_KEY_ESCAPE,              /**< Escape */
+            Enter = GLFW_KEY_ENTER,             /**< Enter */
+            Tab = GLFW_KEY_TAB,                 /**< Tab */
+            Backspace = GLFW_KEY_BACKSPACE,     /**< Backspace */
+            Insert = GLFW_KEY_INSERT,           /**< Insert */
+            Delete = GLFW_KEY_DELETE,           /**< Delete */
+            CapsLock = GLFW_KEY_CAPS_LOCK,      /**< Caps lock */
+            ScrollLock = GLFW_KEY_SCROLL_LOCK,  /**< Scroll lock */
+            NumLock = GLFW_KEY_NUM_LOCK,        /**< Num lock */
+            PrintScreen = GLFW_KEY_PRINT_SCREEN,/**< Print screen */
+            Pause = GLFW_KEY_PAUSE,             /**< Pause */
+            Numpad0 = GLFW_KEY_KP_0,            /**< Numpad 0 */
+            Numpad1 = GLFW_KEY_KP_1,            /**< Numpad 1 */
+            Numpad2 = GLFW_KEY_KP_2,            /**< Numpad 2 */
+            Numpad3 = GLFW_KEY_KP_3,            /**< Numpad 3 */
+            Numpad4 = GLFW_KEY_KP_4,            /**< Numpad 4 */
+            Numpad5 = GLFW_KEY_KP_5,            /**< Numpad 5 */
+            Numpad6 = GLFW_KEY_KP_6,            /**< Numpad 6 */
+            Numpad7 = GLFW_KEY_KP_7,            /**< Numpad 7 */
+            Numpad8 = GLFW_KEY_KP_8,            /**< Numpad 8 */
+            Numpad9 = GLFW_KEY_KP_9,            /**< Numpad 9 */
+            NumpadDecimal = GLFW_KEY_KP_DECIMAL,    /**< Numpad Decimal */
+            NumpadDivide = GLFW_KEY_KP_DIVIDE,      /**< Numpad Divide */
+            NumpadMultiply = GLFW_KEY_KP_MULTIPLY,  /**< Numpad Multiply */
+            NumpadSubtract = GLFW_KEY_KP_SUBTRACT,  /**< Numpad Subtract */
+            NumpadAdd = GLFW_KEY_KP_ADD,        /**< Numpad Add */
+            NumpadEnter = GLFW_KEY_KP_ENTER,    /**< Numpad Enter */
+            NumPadEqual = GLFW_KEY_KP_EQUAL,    /**< Numpad Equal */
```

Comment:
Hmm... both SDL and GLFW name the numpad as `KP`, which might mean ... keypad? Never heard such name :D 

What about shorter prefix, just `Num`? For consistency with "normal" numbers, I would also name the numbers like `NumZero` etc.